src/helpers/client.ts

```diff
@@ -0,0 +1,6 @@
+import { createClient, WithDefaultsT } from "../generated/definitions/client";
+
+export const simpleClient = createClient({
```

Comment:
naming: `BackofficeClient` is more talkative than `simpleClient` (imagine you have multiple API clients)